angular/projects/researchdatabox/form/src/app/form.module.ts

```diff
@@ -18,7 +18,7 @@
 // 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 import { NgModule } from '@angular/core';
 import { BrowserModule, Title } from '@angular/platform-browser';
-import { ReactiveFormsModule } from '@angular/forms';
+import { ReactiveFormsModule, FormsModule } from '@angular/forms';
```

Comment:
We're using reactive forms, not template forms.
The `FormsModule` import should be removed.